### PR TITLE
Add PR tasks to Readme

### PR DESCRIPTION
**Pull request by sid-dahiya**
This change is adding additional instructions to Readme pertaining to Pull Requests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,17 @@ cd build
 ctest -C Debug
 ```
 
+### Additional tasks for Pull Requests
+1. Run `yarn run change`
```
Comment (vmoroz) [inline README.md:79]: >yarn run change [](http://example.com/codeflow?start=8&length=15)

It can be just `yarn change`

File: README.md [hunk 2]
```diff
@@ -75,6 +75,17 @@ cd build
...
 
+### Additional tasks for Pull Requests
+1. Run `yarn run change`
+
+> Tip: If the output of the command is `No change files are needed`
+> once you build with your changes, it is possible beachball 
+> did not pickup your diff correctly. To fix this, run
+> `yarn change --branch "origin/main"` where `origin/main` is
+> your upstream branch.
+2. Ensure the generated `change/*.json` file is present in your
```
Comment (vmoroz) [inline README.md:86]: >2 [](http://example.com/codeflow?start=0&length=1)

Please put an empty string before the next bullet point

File: README.md [hunk 3]
```diff
@@ -75,6 +75,17 @@ cd build
...
+PR.
```
Comment (vmoroz) [inline README.md:87]: Keep it on the previous line?

File: README.md (hunk 1 above)
Comment (vmoroz) [inline README.md:79]: >1 [](http://example.com/codeflow?start=0&length=1)

Please add an empty line before the bullet point

File: README.md [hunk 4]
```diff
@@ -75,6 +75,17 @@ cd build
 ctest -C Debug
 ```
 
+### Additional tasks for Pull Requests
```
Comment (vmoroz) [inline README.md:78]: >### [](http://example.com/codeflow?start=0&length=3)

Should we move the whole new section to the Contributing.md?

Comment (sid-dahiya) [reply README.md]: Sounds good, done.

**Review by vmoroz — APPROVED** [review]
:shipit:

**Pull request closed without merging**